src/vs/workbench/contrib/terminalContrib/suggest/common/terminalSuggestConfiguration.ts

```diff
@@ -201,8 +201,10 @@ export function registerTerminalSuggestProvidersConfiguration(availableProviders
 	const providersProperties: IStringDictionary<IConfigurationPropertySchema> = {};
 
 	const corePwshProviderId = 'core:pwsh-shell-integration';
+	const lspProviderId = 'lsp';
```

Comment:
Because of the `provider: `lsp:${item.extensionId}`` change I don't think this will work anymore? We can hard code the extension name in this config section if needed.

Reply:
<img width="1608" height="819" alt="Screenshot 2025-08-08 at 7 54 55 AM" src="https://github.com/user-attachments/assets/c79090d2-dafa-4625-be1f-438f9b0d34a9" />
Hmm.. I think its still working since we have `readonly id = 'lsp'` in `lspCompletionProviderAddon.ts` 

Should we just hard code this to match  `provider: lsp: extensionId `